base/BUILD.gn

```diff
@@ -3230,16 +3236,15 @@ test("base_unittests") {
     "environment_unittest.cc",
     "feature_list_unittest.cc",
 
-    # "files/file_enumerator_unittest.cc",
+    "files/file_enumerator_unittest.cc",
     "files/file_error_or_unittest.cc",
 
-    # "files/file_path_unittest.cc",
+    "files/file_path_unittest.cc",
 
-    # "files/file_path_watcher_unittest.cc",
-    "files/file_proxy_unittest.cc",
-
-    # "files/file_unittest.cc",
-    # "files/file_util_unittest.cc",
+    "files/file_path_watcher_unittest.cc",
+    # "files/file_proxy_unittest.cc",
```

Comment:
FInd a place to `source -=` this line instead of commenting it out—also why can't it be compiled?